hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/protocolPB/OMPBHelper.java

```diff
@@ -242,11 +242,20 @@ public static MD5MD5Crc32FileChecksumProto convert(
     DataOutputBuffer buf = new DataOutputBuffer();
     checksum.write(buf);
     byte[] bytes = buf.getData();
-    DataInputBuffer buffer = new DataInputBuffer();
-    buffer.reset(bytes, 0, bytes.length);
-    int bytesPerCRC = buffer.readInt();
-    long crcPerBlock = buffer.readLong();
-    buffer.close();
+    DataInputBuffer buffer = null;
+    int bytesPerCRC;
+    long crcPerBlock;
+    try {
+      buffer = new DataInputBuffer();
+      buffer.reset(bytes, 0, bytes.length);
+      bytesPerCRC = buffer.readInt();
+      crcPerBlock = buffer.readLong();
+    } catch (IOException e) {
+      throw new RuntimeException(e);
```

Comment:
Since the method `throws IOException`, I don't think it's necessary to convert it to `RuntimeException`.  (In cases where conversion is needed, prefer the more specific `UncheckedIOException`.)

Reply:
I see. I was not sure the type of the exceptions in this case. Thank you for the advice. I committed the change you suggested (try-with-resources) so I will resolve this conversation.